modules/oe_theme_content_project/config/overrides/core.entity_view_display.node.oe_project.default.yml

```diff
@@ -0,0 +1,199 @@
+    - field.field.node.oe_project.oe_teaser
+    - node.type.oe_project
+  module:
+    - datetime_range
+    - entity_reference_revisions
+    - field_group
+    - link
+    - rdf_skos
+    - text
+    - user
+third_party_settings:
+  field_group:
+    group_top:
+      children:
+        - body
+        - group_details
+        - group_budget
+        - group_resources
+      parent_name: ''
+      weight: 0
+      format_type: html_element
+      region: content
+      format_settings:
+        id: ''
+        classes: ''
+        element: div
+        show_label: false
+        label_element: h3
+        label_element_classes: ''
+        attributes: ''
+        effect: none
+        speed: fast
+        direction: vertical
+      label: Top
+    group_details:
+      children:
+        - oe_reference
+        - oe_project_dates
+      parent_name: group_top
+      weight: 2
+      format_type: oe_theme_helper_field_list_pattern
+      region: content
+      format_settings:
+        variant: horizontal
+        direction: vertical
+        classes: ''
+        id: ''
+      label: Details
+    group_budget:
+      children:
+        - oe_project_budget
+        - oe_project_budget_eu
+        - extra_field_oe_theme_content_project_percentage
+      parent_name: group_top
+      weight: 3
+      format_type: oe_theme_helper_field_list_pattern
+      region: content
+      format_settings:
+        variant: horizontal
+        direction: vertical
+        classes: ''
+        id: ''
+      label: Budget
+    group_resources:
+      children:
+        - oe_project_website
+        - oe_project_departments
+      parent_name: group_top
+      weight: 4
+      format_type: oe_theme_helper_field_list_pattern
+      region: content
+      format_settings:
+        variant: horizontal
+        direction: vertical
+        classes: ''
+        id: ''
+      label: Resources
+id: node.oe_project.default
+targetEntityType: node
+bundle: oe_project
+mode: default
+content:
+  body:
+    label: hidden
+    type: text_default
+    weight: 1
+    settings: {  }
+    third_party_settings: {  }
+    region: content
+  extra_field_oe_theme_content_project_percentage:
+    weight: 9
+    region: content
+    settings: {  }
+    third_party_settings: {  }
+  oe_project_budget:
+    type: number_decimal
+    weight: 7
+    region: content
+    label: above
+    settings:
+      thousand_separator: .
+      decimal_separator: .
+      scale: 0
+      prefix_suffix: true
+    third_party_settings: {  }
+  oe_project_budget_eu:
+    type: number_decimal
+    weight: 8
+    region: content
+    label: above
+    settings:
+      thousand_separator: .
+      decimal_separator: .
+      scale: 0
+      prefix_suffix: true
+    third_party_settings: {  }
+  oe_project_dates:
+    type: daterange_custom
+    weight: 1
+    region: content
+    label: above
+    settings:
+      timezone_override: ''
+      date_format: d.m.Y
```

Comment:
please add format type, see https://github.com/openeuropa/oe_theme/search?q=oe_event_long_date_hour&unscoped_q=oe_event_long_date_hour

Reply:
Should we apply the format on a preprocess instead of config override? Something similar is done on fields  field.field.node.oe_project.oe_project_budget and  field.field.node.oe_project.oe_project_budget_eu.
Is it better to apply these formats on the display configuration or on a preprocess?

Reply:
In the previous link (https://github.com/openeuropa/oe_theme/search?q=oe_event_long_date_hour&unscoped_q=oe_event_long_date_hour) the date format is configured on a config override.

Reply:
yes, export new formats for these fields and apply them with config override